+nosnoc/+discrete_time_problem/Stewart.m

```diff
@@ -582,8 +582,10 @@ function generate_direct_transcription_constraints(obj)
             g_terminal = dcs.g_terminal_fun(x_end, z_end, v_global, p_global);
             relax_terminal_struct = vdx.RelaxationStruct(opts.relax_terminal_constraint.to_vdx, 's_terminal', 'rho_terminal');
             obj.g.terminal = {g_terminal, model.lbg_terminal, model.ubg_terminal, relax_terminal_struct};
+            obj.g.terminal.set_is_terminal(true);
```

Comment:
please add a comment what the method does set_is_terminal? why is it necessary as we already not it is in the terminal constraints group